samples/cellular/lwm2m_carrier/src/main.c

```diff
@@ -23,14 +23,16 @@ static void lte_event_handler(const struct lte_lc_evt *const evt)
 static void on_modem_lib_init(int ret, void *ctx)
 {
 	int err;
-	static bool m_first_init = true;
 
 	ARG_UNUSED(ctx);
 
 	if (ret != 0) {
+		/* Return if modem initialization failed */
 		return;
 	}
 
+	static bool m_first_init = true;
```

Comment:
I don't mind moving this but I get comments sometimes that the variables should be in the beginning of the function. Similar in MOSH.

Reply:
That's ok

Reply:
I completely removed it since it looks like it's not really needed anymore.